exporters/geneva/include/opentelemetry/exporters/geneva/metrics/socket_tools.h

```diff
@@ -306,7 +314,9 @@ struct SocketAddr {
   size_t size() const {
 #ifdef HAVE_UNIX_DOMAIN
     // Unix domain struct m_data_un
```

Comment:
Move this comment to the else if statement below?

Reply:
Thanks this is fixed now.